index.js

```diff
@@ -48,6 +50,11 @@ function Shopify(options) {
     max: undefined
   };
 
+  //
+  // Request timeout.
+  //
+  this.requestTimeout = options.timeout || 60000;
```

Comment:
I wonder if it's better to add a reference to the options object instead of copying the properties to the instance.
So we can use `this.options.timeout` and `this.options.accessToken` below.